.browserslistrc

```diff
@@ -3,11 +3,10 @@
 >= 1%
 last 1 major version
 not dead
-Chrome >= 45
-Firefox >= 38
-Edge >= 12
-Explorer >= 10
-iOS >= 9
-Safari >= 9
-Android >= 4.4
-Opera >= 30
+Chrome >= 70
+Firefox >= 60
```

Comment:
38 was the LTS at the time. Just reminding you.

Reply:
ESR is 60 since May 2018, isn't it?

Reply:
The current ESR version is 60